src/style.css

```
@@ -0,0 +1,46 @@
+* {
+  margin: 0;
+}
+
+/* .container {
+  width: 96px;
+} */
+
+.stars {
+  display: flex;
+  justify-content: start;
+  align-content: center;
+}
+
+.stars__star {
+  background: url(images/star.svg) no-repeat;
+  background-position: left 25% center;
+  width: 16px;
+  height: 16px;
+  padding: 0 2px;
+}
+
+.stars--1 :nth-child(1) {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: left 25% center;
+}
+
+.stars--2 :nth-last-child(n + 4) {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: left 25% center;
+}
+
+.stars--3 :nth-last-child(n + 3) {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: left 25% center;
+}
+
+.stars--4 :nth-last-child(n + 2) {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: left 25% center;
+}
+
+.stars--5 > * {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: left 25% center;
+}
```

you can stack similar styling
and no need to duplicate `background-position: left 25% center;` - you defined it already
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-last-child(n + 4),
.stars--3 :nth-last-child(n + 3),
.stars--4 :nth-last-child(n + 2),
.stars--5 > * {
  background: url(images/star-active.svg) no-repeat;
}
```

---

I added this with shorthand, like this:
background: left 25% center url(images/star-active.svg) no-repeat;
because after apply this suggestion yellow stars moved to top left, 
I think that because of default background-position is top 0 and left 0